### PR TITLE
Os-in-cache-key

### PR DESCRIPTION
**Pull request by jdroenner**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Review by ChristianBeilschmidt — DISMISSED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by jdroenner** [comment]
bors retry

**Comment by jdroenner** [comment]
bors try

**Comment by jdroenner** [comment]
bors retry

**Comment by jdroenner** [comment]
bors try

**Comment by jdroenner** [comment]
bors retry

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by jdroenner** [comment]
[Github Cache für Build-Tesks deaktivieren weil Utoipa-Swagger-Ui im Release-Modus immer PRobleme macht](https://trello.com/c/duJ5CCcY/1055-github-cache-f%C3%BCr-build-tesks-deaktivieren-weil-utoipa-swagger-ui-im-release-modus-immer-probleme-macht)

File: bors.toml [hunk 1]
```diff
@@ -12,6 +12,7 @@ status = [
     "build (ubuntu-22.04, --all-features)",
     "build (ubuntu-22.04, --all-features, --release)",
     "clippy",
+    # "coveralls (ubuntu-22.04, --all-features)",
```
Comment (ChristianBeilschmidt) [inline bors.toml:15]:
```suggestion
```

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**